docs/guide/auto_mode.rst

```diff
@@ -0,0 +1,461 @@
+.. _auto_mode_vehicle_control:
+
+==============================
+Missions (AUTO Mode)
+==============================
+
+AUTO mode is used run pre-defined waypoint missions on Copter, Plane and Rover. 
+
+DroneKit-Python provides basic methods to download and clear the current mission commands 
+from the vehicle, to add and upload new mission commands, to count the number of waypoints, 
+and to read and set the currently executed mission command. 
+Using these primitive methods you can create any other needed mission planning functionality.
```

Comment:
"You can build upon these basic primitives to create high-level mission planning functionality."